master-firmware/src/can/uavcan_node.cpp

```diff
@@ -232,7 +234,7 @@ void main(void *arg)
     static messagebus_topic_t proximity_beacon_topic;
     static MUTEX_DECL(proximity_beacon_topic_lock);
     static CONDVAR_DECL(proximity_beacon_topic_condvar);
-    static float proximity_beacon_topic_value[2];
+    static uint8_t proximity_beacon_topic_value[ProximityBeaconSignal_size];
```

Comment:
The buffer in the protobuf macros is only used for encoding before publishing. You still need one static buffer to hold the topic data.